src/MCPClient/tests/test_xml_metadata.py

```diff
@@ -0,0 +1,570 @@
+#!/usr/bin/env python
```

Comment:
Should this module be renamed to `tests_archivematicaCreateMETSMetadataXML` to hint what's covering?